ketchup/watch_schedule_builder.py

```diff
@@ -43,38 +43,4 @@ def calculate_watch_schedule(
 
     days_available = (end_date - start_date).days
 
```

Comment:
For the lines below this, I want to redo the logic of the code. It's overly complicated. Those permutations we talked about, need to be more streamlined.